test/test_queue.js

```diff
@@ -332,6 +332,15 @@ describe('Queue', function () {
         });
       }, done);
     });
+
+    it('creates a queue with default job options', function (done) {
+      var defaultJobOptions = { removeOnComplete: true };
```

Comment:
where you are at it, why not test also ```removeOnFailed``` ?

Reply:
Sure, happy to add a test for that!